### PR TITLE
Docs (Read the Docs)

### PR DESCRIPTION
**Pull request by lucifurtun**

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```
Comment (ccrisan) [inline docs/Makefile]: I'd remove templated comments like this one.

File: docs/make.bat [hunk 2]
```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```
Comment (ccrisan) [inline docs/make.bat]: We surely don't want to maintain a Windows-based script for generating docs.

File: docs/source/index.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
+.. Colibris documentation master file, created by
```
Comment (ccrisan) [inline docs/source/index.rst]: Can't we not mix rst with md?

File: docs/source/offloading.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# Background Tasks
```
Comment (ccrisan) [inline docs/source/offloading.md]: I'm not against renaming "Background Tasks" to "Offloading" but we should stick with either one or the other, not both.

File: docs/source/testing.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
+# Tests
```
Comment (ccrisan) [inline docs/source/testing.md]: Let's stick with either "Testing" or "Tests" in  both filename and section title.

File: docs/source/web.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Web Server
```
Comment (ccrisan) [inline docs/source/web.md]: Same remark as above - I think it's best to have section name identical to the file name, to the possible extent.

**Comment by ccrisan** [comment]
We should also probably clean up `README.md` before merging this PR.

File: docs/source/index.rst (hunk 3 above)
Comment (lucifurtun) [reply docs/source/index.rst]: Do you think it's a problem to keep them both? I would keep them both until we decide to one format.

Comment (ccrisan) [reply docs/source/index.rst]: I think this is the point where we decide. No point in keeping them both as long as there aren't any technical constraints about them.

Comment (lucifurtun) [reply docs/source/index.rst]: The problem is that I like Markdown but I'm not sure if it's possible to have the index anything other than RST. I don't know if it's worth the effort to rewrite all documentation in RST without being sure that we want RST. I don't have a strong opinion on this issue. What's your proposal?

File: docs/source/Email.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
+# Email Sending
```
Comment (ccrisan) [inline docs/source/Email.md]: Let's have this section renamed to simply "Email". What do you think?

File: docs/source/Health.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+# Health Status
```
Comment (ccrisan) [inline docs/source/Health.md]: "Health" or "Health Status"?

File: docs/source/Initialization.md [hunk 9]
```diff
@@ -0,0 +1,5 @@
+# App Initialization
```
Comment (ccrisan) [inline docs/source/Initialization.md]: "App Initialization" or "Initialization"

File: docs/source/Health.md (hunk 8 above)
Comment (lucifurtun) [reply docs/source/Health.md]: Health

File: docs/source/Email.md (hunk 7 above)
Comment (lucifurtun) [reply docs/source/Email.md]: Ok, "Email" it is.

**Pull request merged**